clients/java/src/main/java/io/camunda/client/impl/command/CreateProcessInstanceCommandImpl.java

```diff
@@ -132,6 +134,19 @@ public CreateProcessInstanceCommandStep3 startBeforeElement(final String element
     return this;
   }
 
+  @Override
+  public CreateProcessInstanceCommandStep3 suspendAfterElement(final String elementId) {
+    grpcRequestObjectBuilder.addRuntimeInstructions(
+        GatewayOuterClass.ProcessInstanceCreationRuntimeInstruction.newBuilder()
+            .setSuspend(
+                SuspendProcessInstanceInstruction.newBuilder()
+                    .setAfterElementId(elementId)
+                    .build()));
```

Comment:
💭 I wasn't aware that our library offers these helper methods to just pass in un-build builders
🔧 Let's do the same in the setSuspend method, just pass in the builder without calling `.build()`. It will be taken care of internally.